tests/python/create.py

```diff
@@ -25,7 +25,7 @@ def parse_args():
     parser.add_argument('--accounts', required=True, type=int, help='Amount of accounts to create')
     parser.add_argument('--passphrase', required=True, type=str, help='Network passphrase')
     parser.add_argument('--horizon', action='append', help='Horizon endpoint URL (use multiple --horizon flags for multiple addresses)')
-
+    parser.add_argument('--json_output', required=False, type=bool, help='Export output to json format')
```

Comment:
Command line parameters should use dashes instead of underscores i.e. `--json-output`.
Only python variables use underscores.

Reply:
also put a default value